DS-Session-1-Assignment/src/question3/MultivariablePolynomial.java

```diff
@@ -0,0 +1,62 @@
+package question3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class represents a multivariable polynomial using nested lists with 
+ * @author Jugal
+ * Dated 07/30/2019
+ */
+public class MultivariablePolynomial {
+	
+	List<Term> multivariablePolynomial = new ArrayList<>();
+	
+	public MultivariablePolynomial(List<Term> termList) {
+		
+		this.multivariablePolynomial = termList;
+	}
+	
+	/**
+	 * Method to calculate the degree of the multivariable polynomial
+	 * @return {int} degree of the multivariable polynomial
+	 */
+	int calculatePolynomialDegree()
+	{
+		int variablePowerSum;
+		int degreeOfPolynomial = 0;
+		for(Term term : multivariablePolynomial)
+		{
+			variablePowerSum = 0;
+			for(Variable variable : term.variableList)
+			{
+				variablePowerSum += variable.getVariablePower();
+			}
+			if(degreeOfPolynomial < variablePowerSum)
+				degreeOfPolynomial = variablePowerSum;
+		}
+		return degreeOfPolynomial;
+	}
+	
+	/**
+	 * This method visualises the Multivariable polynomial into string
+	 * @return {String} Multivariable Polynomial
+	 */
+	String visualiseMultivariablePolynomial()
```

Comment:
First modify the Term and Variable class, after this you may need to change the logic of this method.